docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Organize contracts with workspace
```

Comment:
It's helpful for a reader if titles are brief, because they're displayed in places where a quick glance at a word of two is more accessible to a user.
```suggestion
title: Workspaces
```